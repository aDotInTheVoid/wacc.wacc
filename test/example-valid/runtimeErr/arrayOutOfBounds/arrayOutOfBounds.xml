<unit>
  <main>
    <addr_local>0</addr_local>
    <num>1</num>
    <num>2</num>
    <num>3</num>
    <array_lit>3</array_lit>
    <assign_do/>
    <addr_local>1</addr_local>
    <num>43</num>
    <num>2</num>
    <num>18</num>
    <num>1</num>
    <array_lit>4</array_lit>
    <assign_do/>
    <local>1</local>
    <num>5</num>
    <array_elem/>
    <print><kind>i32</kind><newline>true</newline></print>
  </main>
</unit>