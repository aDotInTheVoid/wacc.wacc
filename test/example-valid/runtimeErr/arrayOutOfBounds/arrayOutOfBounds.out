Index out of bounds: 5, len is 4


---
Exit: 255
